src/content/tags/comp2804/comp2804-a-recursively-defined-set-(4.3).md

```diff
@@ -0,0 +1,6 @@
+---
+course: comp2804
+title: A Recursively Defined Set (4.3)
+number_of_questions: 15
```

Comment:
This value can be calculated at build time. We shouldn't need to store an aggregate